README.md

```diff
@@ -343,10 +345,11 @@ Check the OpenAPI definitions for tasks data endpoints: [Monocle OpenAPI][monocl
 ```YAML
 tenants:
   - index: default
-    task_crawlers:
+    crawlers_api_key: 1a2b3c4d5e
+    crawlers:
       - name: crawler_name
         updated_since: "2020-01-01"
-        api_key: 1a2b3c4d5e
+        provider: TaskDataProvider
```

Comment:
Could we have a complete example for BugZilla ?

Reply:
We could, but this is actually not implemented, we'll need to migrate the lentille-bugzilla first.